presto-yarn-package/src/main/slider/package/templates/config.properties-COORDINATOR.j2

```diff
@@ -4,5 +4,5 @@ discovery-server.enabled=true
 http-server.http.port={{presto_server_port}}
 query.max-memory={{presto_query_max_memory}}
 query.max-memory-per-node={{presto_query_max_memory_per_node}}
-query.queue-config-file={{conf_dir}}/queues.json
+query.max-total-memory-per-node={{presto_query_max_total_memory_per_node}}
```

Comment:
I believe this should also be added here https://github.com/prestodb/presto-yarn/blob/master/presto-yarn-package/src/main/resources/appConfig.json#L15. how will the user configure the property, otherwise?